recipes/ctk-cli/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "ctk-cli" %}
+{%set version = "1.4" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "80bad5eae3ce0d1ece63b53dd2c3dbe4b0bb7f6a18d13748866ae09c83b844b9" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - ctk_cli
+
+about:
+  home: https://github.com/commontk/ctk-cli
+  license: Apache
```

Comment:
With an Apache license, I think we need to include a copy. Since they don't seem package a copy, I would put in an issue upstream, and then make a copy into this recipe and include it here. See for example:

https://github.com/conda-forge/wsaccel-feedstock/blob/master/recipe/meta.yaml